zone/mob.h

```diff
@@ -68,6 +68,7 @@ enum class eSpecialAttacks : int {
 
 class Mob : public Entity {
 public:
+	Timer cheat_timer;
```

Comment:
This needs to be more descriptive, something more like `normal_warp_grace_period_timer`